graphql-dgs-client/src/main/kotlin/com/netflix/graphql/dgs/client/GraphQLClient.kt

```diff
@@ -43,6 +45,19 @@ interface MonoGraphQLClient {
     ): Mono<GraphQLResponse>
 }
 
+interface ReactiveGraphQLClient {
+    fun reactiveExecuteQuery(
+        query: String,
+        variables: Map<String, Any>,
+    ): Publisher<GraphQLResponse>
```

Comment:
As this interface is already exposing Project Reactor types, I think it's safe to return `Flux` here?